packages/compass-aggregations/src/components/pipeline-builder-workspace/pipeline-as-text-workspace/pipeline-preview.tsx

```diff
@@ -115,17 +125,24 @@ export const PipelinePreview: React.FunctionComponent<PipelinePreviewProps> = ({
         {shouldShowCount && <Body>{`Sample of ${docCount} ${docText}`}</Body>}
       </div>
       <PreviewResults isLoading={isLoading} previewDocs={previewDocs} />
```

Comment:
This seems to have some weird padding at the bottom, would be nice not to have it:

![image](https://user-images.githubusercontent.com/5036933/199452241-09abd32d-a8f4-43a8-a629-6251acb54e69.png)

Reply:
I'll be handling that in the next PR to handle Atlas specific operators. If i make changes here, it may create conflicts with my other PR, as I'll have to add classname prop to DocumentList view.